tests/foreman/cli/test_host.py

```diff
@@ -2015,19 +2015,23 @@ def test_negative_without_attach(
 @pytest.mark.cli_host_subscription
 @pytest.mark.tier3
 def test_negative_without_attach_with_lce(
-    target_sat, host_subscription_client, function_org, function_lce
+    target_sat,
+    host_subscription_client,
+    function_org,
+    function_lce,
 ):
     """Attempt to enable a repository of a subscription that was not
-    attached to a host
+    attached to a host.
     This test is not using the host_subscription entities except
     subscription_name and repository_id
 
     :id: fc469e70-a7cb-4fca-b0ea-3c9e3dfff849
 
-    :expectedresults: repository not enabled on host
+    :expectedresults: Repository enabled due to SCA. Why is this "negative"? To keep history, because pre-6.16, this would have failed.
 
     :parametrized: yes
     """
+    function_org.sca_enable()
```

Comment:
SCA will be enabled by default on stream, so we could remove this step?
```suggestion
```

Reply:
Yes, in this place, it really should be removed. We don't expect SCA to be disabled by default ever again.